doc/general/installation/installation.md

```diff
@@ -29,6 +29,7 @@ The following call parameters are available with the launcher `fiskaltrust.exe`:
 | `-sandbox`                     | Sets the environment to be used in the static configuration (`fiskaltrust.exe.config`). The value is a boolean: true (sandbox) \| false (production)            |
 | `-packagesurl`                 | Sets the url of the package server used to download the packages in the static configuration (`fiskaltrust.exe.config`)                                         |
 | `-logfile`                     | Sets the file used to log the output messages in the static configuration (`fiskaltrust.exe.config`)                                                            |
+| `-verbosity`                 | Sets the level of debug-information in the logfile. The value is a string: `Debug`. Use in the static configuration (`fiskaltrust.exe.config`) the key `loglevel` with the value `Debug` |
```

Comment:
We could add the list of possible values:
``Trace`` | ``Debug`` | ``Information`` | ``Warning`` | ``Error`` | ``Critical``

Default is ``Information``

Reply:
Added this! Good hint!